src/turbomind/kernels/decoder_multihead_attention/decoder_multihead_attention_template.h

```diff
@@ -145,7 +145,7 @@ struct DecoderMultiHeadAttentionKernel {
         kv_head_idx_             = head_idx_ / gqa_group_size;
         is_gqa_leader_           = head_idx_ % gqa_group_size == 0;
 
-        timestep_ = params_.per_sample_length[batch_idx_];
+        timestep_ = params_.context_length[batch_idx_] - 1;
```

Comment:
why -1?

Reply:
`context_length` == `sequence_length` + 1, all usage of `sequence_length` are replaced by `context_length` in decoding